dist/backend/web-gl/function-node.js

```diff
@@ -868,14 +890,20 @@ module.exports = function (_FunctionNodeBase) {
 							assumeNotTexture = true;
 						}
 					}
-
+					debug_log("- astMemberExpression " + reqName + " " + funcName);
 					if (assumeNotTexture) {
 						// Get from array
 						this.astGeneric(mNode.object, retArr);
 						retArr.push('[int(');
 						this.astGeneric(mNode.property, retArr);
 						retArr.push(')]');
 					} else {
+						var multiMemberExpression = this.isState('multi-member-expression');
+						if (multiMemberExpression) {
+							this.popState('multi-member-expression');
+							this.pushState('not-in-get-call-parameters');
```

Comment:
You are pushing a state that is never checked for, so is this actually needed?

Reply:
yes it is! We need the state that we are no longer currently in the 'in-get-parameters' state ( I was thinking of calling it the force-int state which might help with your understanding - but probably too generic for the actual lib).

any call to this.astGeneric will check the state and nested gets will have issues without this...